README.md

```diff
@@ -171,6 +165,23 @@ PayPal used to support authentication via a modal iframe, but authentication now
 
 PopupBridge solves this problem by allowing [`braintree-web`](https://github.com/braintree/braintree-web) to open the PayPal popup from a secure mini-browser.
 
+Using PayPal in a WebView
+-------------------------
+
+WebView-based checkout flows can accept PayPal with PopupBridge the Braintree JS SDK. For the authentication flow, PayPal requires a popup window—which can be simulated with PopupBridge.
+
+### Setup
+1. Create a web-based checkout that accepts PayPal using Braintree JS v3.9.0 or higher
+2. Show a loading indicator
+    - PopupBridge lacks the PayPal loading page that customers see when using PayPal through a real popup window. We recommend adding a loading indicator on your web page when the PayPal button is clicked. In the PayPal tokenize completion callback, hide the loading indicator.
+3. Create a native mobile app that opens the checkout in a WebView
+4. Integrate the PopupBridge library
+5. Collect device data
+    - To help detect fraudulent activity, collect device data before performing PayPal transactions. This is similar to collecting device data with our [native Android SDK](https://developers.braintreepayments.com/guides/paypal/vault/android/v2#collecting-device-data) with a few differences:
+        1. Rather than the entire data collector, you can add just PayPalDataCollector to your app, e.g. `compile 'com.paypal.android.sdk:data-collector:2.4.2'`
```

Comment:
The docs are wrong on https://developers.braintreepayments.com/guides/paypal/vault/android/v2#collecting-device-data and should not include `com.braintreepayments.api:data-collector`. Reword this to Include `PayPalDataCollector` in your `build.gradle` dependencies.